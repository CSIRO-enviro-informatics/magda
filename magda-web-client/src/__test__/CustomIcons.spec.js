import React from 'react';
import renderer from 'react-test-renderer';
import CustomIcons from '../UI/CustomIcons';

it('renders default icon', () => {
  const tree = renderer.create(
    <CustomIcons/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});


it('renders jpeg icon', () => {
  const tree = renderer.create(
    <CustomIcons name='JPG' className='test'/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});


it('renders custom url icon', () => {
  const tree = renderer.create(
    <CustomIcons imageUrl='http://lorempixel.com/400/200/'/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
