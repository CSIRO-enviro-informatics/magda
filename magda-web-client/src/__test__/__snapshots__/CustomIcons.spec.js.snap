// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders custom url icon 1`] = `
<img
  alt={undefined}
  className={undefined}
  src="http://lorempixel.com/400/200/"
/>
`;

exports[`renders default icon 1`] = `
<img
  alt={undefined}
  className={undefined}
  src="/assets/file-icons/Default.png"
/>
`;

exports[`renders jpeg icon 1`] = `
<img
  alt="JPG"
  className="test"
  src="/assets/file-icons/Default.png"
/>
`;
